public/pages/DetectorResults/utils/constants.ts

```diff
@@ -13,12 +13,15 @@
  * permissions and limitations under the License.
  */
 
-import { SORT_DIRECTION } from '../../../../server/utils/constants';
+import {
+  SORT_DIRECTION,
+  AD_DOC_FIELDS,
+} from '../../../../server/utils/constants';
 
 export const DEFAULT_QUERY_PARAMS = {
   from: 0,
   search: '',
-  size: 20,
+  size: 30,
```

Comment:
Question: Any specific reason on getting 30 v/s 20 ?

Reply:
I think I changed it by mistake. I need 30 anomaly results as the Live Chart needs results within last 30 mins, so we need at most 30 results given the smallest interval - 1 mins. Will revert it back to 20.